puppet-squid/src/main/java/com/iadams/sonarqube/puppet/api/PuppetGrammar.java

```diff
@@ -409,10 +409,12 @@ public static void compoundStatements(LexerfulGrammarBuilder b) {
       b.zeroOrMore(STATEMENT),
       RBRACE,
       b.zeroOrMore(ELSEIF_STMT),
-      b.optional(ELSE, LBRACE, b.zeroOrMore(STATEMENT), RBRACE));
+      b.optional(ELSE_STMT));
```

Comment:
@iwarapter Update https://github.com/iwarapter/sonar-puppet/issues/157 to deal with `if` statements nested in `else` statements.